### PR TITLE
Completed Search Insert Position 

### PR DESCRIPTION
**Pull request by bagladivyang03**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by bagladivyang03** [comment]
@plazzy99 @Kajol-Kumari @HarshCasper @siddharth25pandey @Abhijit2505 @iamrajiv  pls review and merge asap:)

File: C-Plus-Plus/cp/search_insert_position.cpp [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+You may assume no duplicates in the array.
+*/
+#include <iostream>
+using namespace std;
+
+int search_insert_position(int arr[], int n, int target){
+    int l=0,r=n-1;
+    int mid;
+    while(l<=r){
+        mid = (l+r)/2;
```
Comment (iamrajiv) [inline C-Plus-Plus/cp/search_insert_position.cpp]: @bagladivyang03 Use `mid = l + ((h - l) / 2)` to avoid integer overflow condition.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@bagladivyang03 Add space and time complexity.

**Comment by bagladivyang03** [comment]
> @bagladivyang03 Add space and time complexity.

In comments should I write?

**Comment by iamrajiv** [comment]
@bagladivyang03 Yes.

**Comment by bagladivyang03** [comment]
@iamrajiv pls check now :)

**Review by iamrajiv — APPROVED** [review]

**Comment by bagladivyang03** [comment]
@plazzy99 @atarax665 pls review asap and merge :)

**Review by vatsalkesarwani12 — APPROVED** [review]
👍🏻

**Comment by bagladivyang03** [comment]
@atarax665 pls review and merge asap :)

**Comment by bagladivyang03** [comment]
@atarax665 pls merge and review as tomorrow is the last date pls :)

**Pull request merged**